Core/WebServer.cpp

```diff
@@ -49,6 +51,16 @@ static ServerStatus serverStatus;
 static std::mutex serverStatusLock;
 static int serverFlags;
 
+// NOTE: These *only* encode spaces, which is really enough.
```

Comment:
Well, you also want to ecnode at least %.  A filename with "%20" in it (such as a downloaded file) will not work.

-[Unknown]

Reply:
Yeah, true. Just doing what the old Recent code does.

Reply:
Oh yeah, and I did first use UriDecode, but some logic broke because it encoded slashes. Should maybe fix the root cause of that instead maybe, but could also simply not encode slashes to keep compatibilty with older versions.